sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseDataForm.java

```diff
@@ -1193,7 +1193,9 @@ private void updateFacilityDetails(ComboBox cbFacility, TextField tfFacilityDeta
 			boolean visibleAndRequired = otherHealthFacility || noneHealthFacility;
 
 			tfFacilityDetails.setVisible(visibleAndRequired);
-			tfFacilityDetails.setRequired(visibleAndRequired);
+			if (otherHealthFacility) {
+				tfFacilityDetails.setRequired(visibleAndRequired);
```

Comment:
this is a bit confusing, the variable name is a bad one as it represents two things and is already defined by the boolean in the if statement.. please consider cleaning this up

Reply:
updated, removed if clause and linked setRequired to "otherHealthFacility"